### ISSUE TITLE
false positive "'elseif' is not needed here"

### ISSUE THREAD

**Issue by eigan** [kalessil/phpinspectionsea#768]
Version: 2.3.16
PHPStorm: 2017.3.3

Code example:

```php
if ($i === 0 && $a === $b) {
    return '';
} elseif ($i === 0 /* && $a and $b is not the same */) {
    return '';
}

//  [...]
```

`elseif` is still needed.

**Comment by kalessil** [comment]
Thank you for reporting @eigan. It's intended to use `if` instead of `elseif` here, we have explained why earlier: #426 #444,
